examples/rbd/secret.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
   name: csi-rbd-secret
   namespace: default
 data:
-  # Key value corresponds to a user name defined in ceph cluster
-  admin: BASE64-ENCODED-PASSWORD
-  # Key value corresponds to a user name defined in ceph cluster
-  kubernetes: BASE64-ENCODED-PASSWORD
+  # Key values correspond to a user name and its key, base 64 encoded, as
+  # defined in the ceph cluster. User ID should have required access to the
+  # 'pool' specified in the storage class
+  userID: BASE64-ENCODED-VALUE
```

Comment:
The ID shouldn't need to be base64 encoded

Reply:
If the key `userID` is mentioned in a section named `stringData` it need not be encoded in base64 and can be mentioned in plain text. This is how a secret is specified in Kubernetes [1].

If required I can change the documentation to reflect the added `stringData` and mentioning the `userID` key within the same.

[1] Kubernetes manual secrets and `data` `stringData` options for base64 and plain text secrets: https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually

Reply:
I'd vote for ```stringData``` --- honestly, since the key is already base64 encoded, I always thought it was a little silly (usability-wise) that it needed to be encoded again, so perhaps both could be moved? or perhaps just move the ```userId``` back to the ```StorageClass```?

Reply:
Then `stringData` it is. I will move both, as the key otherwise is double base64 encoded, as you point out.

Fractured ID and key in 2 files, to me, is a far worse usability proposition, so no I would not go that way.